### PR TITLE
fix for FHIR-46147

### PR DESCRIPTION
**Pull request by dt-r**
Change all instances of "how to search by logical id" in Notes to "how to search by id of the resource" and hyperlink to https://hl7.org/fhir/r4/search.html#id

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-notes.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ The following search parameters and search parameter combinations **SHOULD** be
       1. GET [base]/MedicationRequest?_id=2169591
       1. GET [base]/MedicationRequest?_id=2169591&amp;_include=MedicationRequest:medication
 
-    *Implementation Notes:* Fetches a single MedicationRequest ([how to search by the logical id](http://hl7.org/fhir/R4/references.html#logical) of the resource)
+    *Implementation Notes:* Fetches a single MedicationRequest ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id) of the resource)
```
Comment (JayMurdoch) [inline input/intro-notes/StructureDefinition-au-core-medicationrequest-notes.md:133]: Remove duplicate "of the resource"

File: input/intro-notes/StructureDefinition-au-core-organization-notes.md [hunk 2]
```diff
@@ -82,5 +82,5 @@ The following search parameters **SHOULD** be supported:
     
       1. GET [base]/Organization?_id=5678
 
-    *Implementation Notes:* Fetches a single Organization ([how to search by the logical id](http://hl7.org/fhir/R4/references.html#logical) of the resource)
+    *Implementation Notes:* Fetches a single Organization ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id) of the resource)
```
Comment (JayMurdoch) [inline input/intro-notes/StructureDefinition-au-core-organization-notes.md:85]: Remove duplicate "of the resource"

File: input/intro-notes/StructureDefinition-au-core-patient-notes.md [hunk 3]
```diff
@@ -94,7 +94,7 @@
       1. GET [base]/Patient/5678
       1. GET [base]/Patient?_id=5678
 
-    *Implementation Notes:* Returns a single Patient resource ([how to search by the logical id](http://hl7.org/fhir/R4/references.html#logical) of the resource)
+    *Implementation Notes:* Returns a single Patient resource ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id) of the resource)
```
Comment (JayMurdoch) [inline input/intro-notes/StructureDefinition-au-core-patient-notes.md:97]: Remove duplicate "of the resource"

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ The following search parameters **SHOULD** be supported:
       1. GET [base]/Practitioner/5678
       1. GET [base]/Practitioner?_id=5678
 
-    *Implementation Notes:* Returns a single Practitioner resource ([how to search by the logical id](http://hl7.org/fhir/R4/references.html#logical) of the resource)
+    *Implementation Notes:* Returns a single Practitioner resource ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id) of the resource)
```
Comment (JayMurdoch) [inline input/intro-notes/StructureDefinition-au-core-practitioner-notes.md:58]: Remove duplicate "of the resource"

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md [hunk 5]
```diff
@@ -91,7 +91,7 @@ The following search parameters and search parameter combinations **SHOULD** be
       1. GET [base]/PractitionerRole/5678
       1. GET [base]/PractitionerRole?_id=5678&amp;_include=PractitionerRole:practitioner
 
-    *Implementation Notes:* Returns a single PractitionerRole resource. ([how to search by the logical id](http://hl7.org/fhir/R4/references.html#logical) of the resource)
+    *Implementation Notes:* Returns a single PractitionerRole resource. ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id) of the resource)
```
Comment (JayMurdoch) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md:94]: Remove the duplicate "of the resource"

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Hi Danielle,
There's a duplicate 'of the resource' to be removed from all changes.

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-notes.md (hunk 1 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-medicationrequest-notes.md]: fixed

File: input/intro-notes/StructureDefinition-au-core-organization-notes.md (hunk 2 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-organization-notes.md]: fixed

File: input/intro-notes/StructureDefinition-au-core-patient-notes.md (hunk 3 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-patient-notes.md]: fixed

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md (hunk 4 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitioner-notes.md]: fixed

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md (hunk 5 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md]: fixed

**Review by JayMurdoch — APPROVED** [review]
Changes reviewed, no further issues.

**Pull request merged**